gammapy/spectrum/fit.py

```diff
@@ -204,8 +204,11 @@ def _predict_counts_helper(self, obs, model, forward_folded=True):
                                               e_reco=binning)
             counts = temp.data.data
         else:
-            # TODO: This could also be part of calculate predicted counts
-            counts = model.integral(binning[:-1], binning[1:])
+            temp = calculate_predicted_counts(model=model,
```

Comment:
The ``forward_folding=False`` flag was meant for the case, when no IRFs are given at all (i.e. the model is directly in counts space). The way you changed it, now forsees the case that only an effective area is given, right? This should have worked before by setting ``forward_folding=True`` and ``edisp=None``. Didn'it it work for you? If it worked before, we should probably just improve the documentation. If not, we have to discuss again.